starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildCache.rocker.raw

```diff
@@ -0,0 +1,34 @@
+@import io.micronaut.starter.feature.build.gradle.BuildRemoteCacheConfiguration
+@import io.micronaut.starter.build.gradle.GradleDsl
+@args (GradleDsl dsl, BuildRemoteCacheConfiguration config)
+@if(dsl == GradleDsl.KOTLIN) {
+buildCache {
+    remote<HttpBuildCache> {
+        url = uri("@(config.getUrl())")
+@if(config.push()) {
+        isPush = true
+}
+@if(config.getCredentials() != null) {
+        credentials {
+            username = "@(config.getCredentials().getUsername())"
```

Comment:
Probably we want to recommend that the username "value" is actually something like `System.getProperty("...")`, so that the script doesn't contain the full username/password. In this case it probably also makes sense to escape the value so that quotes are properly rendered.

Reply:
See https://github.com/micronaut-projects/micronaut-starter/pull/1048/commits/8e03dcb8ed46edbd0bbd2b9ee85f911f294b47d6 I think that should resolve your concerns. If it does, please resolve this conversation.